### PR TITLE
Update set up multicluster guide

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md [hunk 1]
```diff
@@ -119,14 +135,28 @@ The component [Tower](https://github.com/kubesphere/tower) of KubeSphere is used
...
 
-* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as below: 
+* In order for host cluster to manager member cluster, we need to make the jwtSecret same between them. So first get it from the host by the following command.
+
+    ```bash
+    $ kubectl -n kubesphere-system get cm kubesphere-config -o yaml | grep -v "apiVersion" | grep jwtSecret
+      jwtSecret: "gfIwilcc0WjNGKJ5DLeksf2JKfcLgTZU"
+    ```
+
+* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following, input the jwtSecret we have from the step above and wait for the installation to be completed.
```
Comment (FeynmanZhou) [inline sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md:145]: `input the jwtSecret we have from the step above` => `input the corresponding jwtSecret shown above`

File: sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md [hunk 2]
```diff
@@ -37,21 +37,37 @@ if the kube-apiserver address of M Cluster is accessible on any node of the H Cl
...
 
-* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following and wait for the installation to be completed.
+* In order for host cluster to manager member cluster, we need to make the jwtSecret same between them. So first get it from the host by the following command.
+
+    ```bash
+    $ kubectl -n kubesphere-system get cm kubesphere-config -o yaml | grep -v "apiVersion" | grep jwtSecret
+      jwtSecret: "gfIwilcc0WjNGKJ5DLeksf2JKfcLgTZU"
+    ```
+
+* There is no difference between the installation of Member Cluster and the installation of common clusters that have their multi-cluster feature disabled. Please make sure "multicluster" is set as the following, input the jwtSecret we have from the step above and wait for the installation to be completed.
```
Comment (FeynmanZhou) [inline sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md:55]: `input the jwtSecret we have from the step above` => `input the corresponding jwtSecret shown above`

File: sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md (hunk 1 above)
Comment (rayzhou2017) [reply sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md:145]: `as the following` is same as `as follows`. I am going to update the doc based on the second comment

Comment (FeynmanZhou) [reply sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md:145]: Ref: https://english.stackexchange.com/questions/68376/as-follows-vs-as-the-following

Comment (rayzhou2017) [reply sig-multicluster/how-to-setup-multicluster-on-kubesphere/README.md]: Ok, I changed it. But `as the following` should appear everywhere in textbook.

**Pull request merged**